CHANGELOG.md

```diff
@@ -1,7 +1,26 @@
-## next release
+## 2.3.0 / 2018-06-05
 
```

Comment:
There's a password_file for basic auth now too

/metrics names for HTTP stuff are changing

Reply:
Added